### ISSUE TITLE
[Accessibility]: Text contrast of 1.96 with its background is less than the WCAG AA minimum requirements for text of size 12px and weight of 400

### ISSUE THREAD

**Issue by yashgarg1719** [carbon-design-system/carbon-charts#1407]
### Contact Details

yashgarg@kyndryl.com

### Environment

Browser, Automated testing tool and ruleset, Assistive technology used to verify

### Issue
Text contrast of 1.96 with its background is less than the WCAG AA minimum requirements for text of size 12px and weight of 400

### Expectation

The contrast ratio of text with its background must meet WCAG 2.1 AA requirements

When text and its background colors have less than a 4.5 to 1 contrast ratio it can be difficult for people with moderately low vision to read the text without a contrast-enhancing technology. For larger text of 18 point or more, or bold 14 point text, the text and background colors must give at least a 3 to 1 contrast ratio.

For large text that is at least 18 point or at least 14 point when bold, adjust the color of the text or its background to give at least a 3 to 1 contrast ratio; Or, for body text that is less than 18 point or less than 14 point when bold, adjust the color of the text or its background to give at least a 4.5 to 1 contrast ratio.

People with moderately low vision who do not use a contrast-enhancing technology People with visual impairment using color contrast enhancement People who cannot see color, have limited color vision or have color blindness.

Rule ID: IBMA_Color_Contrast_WCAG2AA Reason

### Testing environment details
Operating System: MacOS
Browser: Firefox 101.0(64-bit)

### Screenshots

![image](https://user-images.githubusercontent.com/108860673/178149869-dc82a09f-a9c6-4d68-a593-34d4b50bc643.png)

### What WCAG 2.1 checkpoint does the issue violate?

IBM 1.4.3 Contrast (Minimum) WCAG technique G18 WCAG technique G145

### Version

`@carbon/charts@0.41.79`

**Comment by mbgower** [comment]
@yashgarg1719 can you please provide the steps to reproduce this? Provide a URL and any theme settings, etc., you change to produce this result. Thanks!

**Comment by mbgower** [comment]
@theiliad, @thefirstartist was talking about a similar error in charts but the text in the axis he was demonstrating was actually much better contrast although the checker was flagging the issue. Maybe Sam can provide his steps to reproduce here as well, so we can determine if it is the same underlying problem.

**Comment by thefirstartist** [comment]
@mbgower @theiliad Here's the issue I logged: https://github.com/carbon-design-system/carbon-charts/issues/1463
I have included screenshots showing how I reproduced the issue using storybook.

https://github.com/carbon-design-system/carbon-charts/issues/1463

**Comment by theiliad** [comment]
Going to close this out, I'm not able to see this violation using the scanner anymore.
